CHANGELOG.md

```diff
@@ -31,6 +31,25 @@
   export SENTRY_DISABLE_NATIVE_DEBUG_UPLOAD=true
   ```
 
+- Add Replay Custom Masking for iOS ([#4238](https://github.com/getsentry/sentry-react-native/pull/4224))
+
+  ```jsx
+  import * as Sentry from '@sentry/react-native';
+
+  const Example = () => {
+    return (
+      <View>
+        <Sentry.Mask>
```

Comment:
Q: for now, <Sentry.Mask> / <Sentry.Unmask> are exclusive to native/ React Native?

Reply:
This PR adds the component impl for iOS, next PR adds it for Android and next one for RN Web.